packages/components/package.json

```diff
@@ -51,6 +51,7 @@
     "ariakit": "^2.0.0-next.43",
     "html-react-parser": "^3.0.9",
     "lodash": "^4.17.21",
+    "lucide-react": "^0.218.0",
```

Comment:
Should this be in both `dependencies` and `peerDependencies`? At least based on how `@heroicons/react`, not sure if that's actually needed.

Reply:
Ah, my bad!